test/ImageEnhanceViaFusion_test.cpp

```diff
@@ -0,0 +1,93 @@
+
+#include <string>
+#include <iostream>
+#include <vector>
+
+#include <boost/test/test_tools.hpp>
+#include <boost/test/unit_test_suite.hpp>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+
+#include <underwater_image_tools/ImageEnhanceViaFusion.hpp>
+
+
+#define PATH_DATASET "../test/dataset/"
```

Comment:
Try to use this as absolute path, for example:

<pre>
std::string current_path(__FILE__);
current_path = current_path.substr(0, current_path.find_last_of("/"));
</pre>